docs/cookbook.rst

```diff
@@ -80,10 +82,11 @@ Starlette. You can add it to your application, ideally in front of the ``Routing
         :sync: ConnexionMiddleware
 
         .. code-block:: python
+            from pathlib import Path
 
             from asgi_framework import App
             from connexion import ConnexionMiddleware
-            from connexion.lifecycle import ConnexionRequest, ConnexionResponse
```

Comment:
Unless there's a hidden reason to import these classes (?) they were not used in the snippet.